index.js

```diff
@@ -4,7 +4,35 @@
  * Depending on the operation, either add up all of the numbers or subtract all of the numbers, from left to right.
  * @returns {number} The result of either adding all numbers or subtracting all numbers, depending on the arguments added to the command line.
  */
-function calculator() {}
+function calculator() {
+    let command = process.argv[2];
+    let args = process.argv.length;
+    let sum = 0;
+    //checking for errors;
+    if(!command){
```

Comment:
If i run `node run.js 1 2 3`, I get `Invalid operation: 1`. How can you update this conditional to make sure you get the correct error message?

Reply:
Well, in that case process.argv[2] is equaling a number. So I would have to check if that argument is a number a deal with it accordingly.